tests/test_code_execution.py

```diff
@@ -223,10 +223,10 @@ def test_few_shots(sandbox_type, code_begin, code_end, code_output_begin, code_o
                 session_id = None
                 for code_snippet, expected_output in zip(code_snippets, expected_outputs):
                     output, session_id = sandbox.execute_code(code_snippet, session_id=session_id)
+                    execution_dict = {"process_status": "completed", "stdout": expected_output.strip(), "stderr": None}
```

Comment:
Do we have to do .strip()?

Reply:
Yes, because `extract_code_output` returns everything between `code_output_begin` and `code_output_end` including `\n`'s while `sandbox.execute_code` returns just code output without any `\n`'s